app/helpers/application_helper.rb

```diff
@@ -1,4 +1,27 @@
 module ApplicationHelper
+  def analytics_body_tag(attributes = {}, &block)
```

Comment:
Worth a test here? You could probably dynamically load the controller names in a spec and assert they appear in the `controller` attribute so it fails if someone adds a new analytics controller but forgets to include it here/ends up adding it manually instead of using the helper